.gitignore

```diff
@@ -8,6 +8,7 @@ cscope.po.out
 cscope.out
 smf/manifests/*.xml
 .DS_Store
+.idea
```

Comment:
This should be in your global gitignore, not in the project: https://github.com/github/gitignore/blob/master/Global/JetBrains.gitignore